apps/web/src/components/templates/email-editor/EmailMessagesCards.tsx

```diff
@@ -85,13 +85,14 @@ export function EmailMessagesCards({ index, isIntegrationActive }: { index: numb
                     width: '70px',
                   },
                   active: {
-                    background: theme.colorScheme === 'dark' ? colors.white : colors.B98,
+                    background: theme.colorScheme === 'dark' ? colors.B40 : colors.B98,
                     borderRadius: '30px',
                   },
                   labelActive: {
                     color: `${colors.B40} !important`,
```

Comment:
shouldn't we apply similar color styles here as in the preview switch?

Reply:
can you please provide more details, I am not getting

Reply:
@BhuvaneshPatil what he meant is that in line 92 you should apply the same change as in line 29 of `apps/web/src/components/templates/EditorPreviewSwitch.tsx`.

Reply:
Thank @p-fernandez , I'll update and push the new changes